app/views/contacts/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>My Contacts</h1>
+
+<% if user_signed_in? %>
+  <table>
+    <thead>
+    <tr>
+      <th>Email</th>
+    </tr>
+    </thead>
+    <tbody>
+    <% current_user.contacts.each do |contact| %>
```

Comment:
I am not sure about it but do you think it is a good way to call the database from a view? I am just thinking about it since we are doing the same thing in der home view?

Reply:
We could maybe in the controller something like: `@contacts = current_user.contacts` in the show action and then we can use @contacts here in the view

Reply:
was improved